docs/book/src/reference/using_an_external_type.md

```diff
@@ -0,0 +1,303 @@
+
+Note that if you did not pass `--domain` to `kubebuilder init` you will need to modify it for the individual api types as the default is `my.domain`, not `theirs.com`.
+Similarly, if you intend to use your own domain, please configure your own domain with `kubebuilder init` and do not use `theirs.com for the domain.
+
+### Add a controller for the external Type
+
+#### Using a type not included in the Kubernetes Core Resources
+
+We will now add a controller for the given type
+
+```shell
+kubebuilder create api --group theirgroup --version v1alpha1 --kind ExternalType --controller --resource=false
+```
+
+Note that the `resource` argument is set to false, as we are not attempting to create our own CustomResourceDefinition,
+but instead rely on an external one.
+
+This will result in a `PROJECT` entry with the default domain of the `PROJECT` (`my.domain` if not specified in `kubebuilder init`).
+For use of other domains, such as `theirs.com`, one will have to manually adjust the `PROJECT` file with the correct domain for the entry:
+
+file: PROJECT
+```
+domain: my.domain
+layout:
+- go.kubebuilder.io/v4
+projectName: testkube
+repo: github.com/jakobmoellerdev/testkube
+resources:
+- controller: true
+  domain: theirs.com
+  group: mygroup
+  kind: ExternalType
+  version: v1alpha1
+version: "3"
+```
+
+At the same time, the generated RBAC manifests need to be adjusted:
+
+file: internal/controller/externaltype_controller.go
+```go
+// ExternalTypeReconciler reconciles a ExternalType object
+type ExternalTypeReconciler struct {
+	client.Client
+	Scheme *runtime.Scheme
+}
+
+//+kubebuilder:rbac:groups=theirgroup.theirs.com,resources=externaltypes,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=theirgroup.theirs.com,resources=externaltypes/status,verbs=get;update;patch
+//+kubebuilder:rbac:groups=theirgroup.theirs.com,resources=externaltypes/finalizers,verbs=update
+```
+
+#### Using a type included in the Kubernetes Core Resources
+
+Alternatively, one can also use [core resources](https://github.com/kubernetes-sigs/kubebuilder/blob/c172069e83d81cc4a6ce67a7a1c3fd2a6e7e2d00/pkg/plugins/golang/options.go#L28-L54), such as Pods, as a controlled resource:
+
+```shell
+kubebuilder create api --version v1 --kind Pod --controller --resource=false
+```
+
+Similarly, the default domain will be used, so it has to be corrected:
+
+file: PROJECT
+```
+domain: my.domain
+layout:
+- go.kubebuilder.io/v4
+projectName: testkube
+repo: github.com/jakobmoellerdev/testkube
+resources:
+- controller: true
+  domain: ""                                                                                                                                                                                             
+  kind: Pod
+  version: v1alpha1
+version: "3"
```

Comment:
I would like to suggest we remove this part and add it at the end add a note like:

```
If you are looking to create Controllers to manage Kubernetes Core types (i.e. Deployments/Pods):

Then you do not need to update the PROJECT file or register the Schema in the manager. All Core Types are registered by default. The Kubebuilder CLI will add the required values to the PROJECT file, but you still need to perform changes to the RBAC markers manually to ensure that the Rules will be generated accordingly.
```